tests/Microsoft.Identity.Test.Core.UIAutomation/MobileTestHelper.cs

```diff
@@ -282,13 +290,29 @@ public void SetAuthority(ITestController controller, string authority)
             // Select authority
             controller.Tap(CoreUiTestConstants.AuthorityPickerId);
             controller.Tap(authority);
+
+            if (_platform == Xamarin.UITest.Platform.iOS)
+            {
+                controller.Tap("Done");
+            }
         }
 
         public void PerformB2CLocalAccountSignInFlow(ITestController controller, LabUser user, UserInformationFieldIds userInformationFieldIds)
         {
             controller.EnterText(CoreUiTestConstants.WebUpnB2CLocalInputId, 20, user.Upn, XamarinSelector.ByHtmlIdAttribute);
 
-            controller.EnterText(userInformationFieldIds.GetPasswordInputId(true), user.GetOrFetchPassword(), XamarinSelector.ByHtmlIdAttribute);
+            if (_platform == Xamarin.UITest.Platform.iOS)
+            {
+                controller.Tap(CoreUiTestConstants.B2CWebPasswordId, XamarinSelector.ByHtmlIdAttribute);
+                Thread.Sleep(1000);
```

Comment:
magic

Reply:
this was advised by support, and this test is currently ignored, but I will remove the waits for now.

---
In reply to: [296153527](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1226#discussion_r296153527) [](ancestors = 296153527)